packages/cerebral-provider-firebase/src/sendPasswordRestEmail.js

```diff
@@ -0,0 +1,14 @@
+import firebase from 'firebase'
+
+export default function sendPasswordResetEmail (email) {
```

Comment:
File name does not match function name. "sendPasswordResetEmail" (missing "e" in Reset)...